### DISCUSSION TITLE
Character Creation - Final step

### DISCUSSION THREAD

**Discussion by JeremyTHolland** [DavidRieman/WheelMUD#51] [Ideas]
When you're done its a little unclear what to do next after the message:

> You enter the world.
> Welcome, [name]

Maybe showing the room description would be helpful there just to give some context and location information?

**Comment by DavidRieman** [comment]
True.  At one point, IIRC we just forced a "look" command for the user, but the way we did it was awkward and isn't supported anymore.  I agree, for now upon login we could probably just carefully ensure the user's command queue starts emptied but with one "look" command inserted.  This can become an Issue for our backlog, or you can try to tackle it if you want.

This discussion spawned #56
